libsplinter/src/peer/connector.rs

```diff
@@ -44,7 +44,11 @@ pub trait PeerLookup: Send {
     fn peer_id(&self, connection_id: &str) -> Result<Option<String>, PeerLookupError>;
 }
 
+/// The `PeerLookupProvider` trait facilitates getting the peer IDs and connection IDs for
+/// messages.
 pub trait PeerLookupProvider {
+    /// Returns a Box<dyn PeerLookup> to be used for getting a connection ID from the associated
```

Comment:
`Box<dyn PeerLookup>` should be within ``

Reply:
fixed